### ISSUE TITLE
Every time there's a new version, it pops up after you boot it up, and it feels bad

### ISSUE THREAD

**Issue by MOOA-code** [QL-Win/QuickLook#718]
Please solve it.
Thanks.

**Comment by xupefei** [comment]
Are you on an old version and do not want to upgrade?

**Comment by MOOA-code** [comment]
Yes, please don't let it automatically popover, a version can actually be
used for a long time, thank you

Paddy Xu <notifications@github.com> 于2020年8月9日周日 下午10:43写道：

**Comment by MOOA-code** [comment]
I think it’s best not to pop up once, most of the software is like this

Paddy Xu <notifications@github.com> 于2020年8月22日周六 下午6:27写道：
